apps/forklift/lib/forklift/data_writer.ex

```diff
@@ -113,15 +114,16 @@ defmodule Forklift.DataWriter do
   defp do_write(data, dataset) do
     started_data = Enum.map(data, &add_start_time/1)
 
-    retry with: exponential_backoff(100) |> cap(@max_wait_time) do
+    retry with: exponential_backoff(@retry_count) |> expiry(@max_wait) do
```

Comment:
I think we misunderstood this when we edited it. We ultimately want to either set a max number of retries or a max total time so our tests can actually fail. @jdenen - the `cap` usage in this case, was to keep the exponential backoff from growing forever, right?

Reply:
This is fixed.